modules/storages/lib/api/v3/storages/storage_authorizer.rb

```diff
@@ -0,0 +1,43 @@
+#-- copyright
+# OpenProject is an open source project management software.
+# Copyright (C) 2012-2022 the OpenProject GmbH
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License version 3.
+#
+# OpenProject is a fork of ChiliProject, which is a fork of Redmine. The copyright follows:
+# Copyright (C) 2006-2013 Jean-Philippe Lang
+# Copyright (C) 2010-2013 the ChiliProject Team
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+#
+# See COPYRIGHT and LICENSE files for more details.
+#++
+
+module API
+  module V3
+    module Storages
+      class StorageAuthorizer
+        class << self
+          def authorize(storage)
+            oauth_client = storage.oauth_client
+            connection_manager = ::OAuthClients::ConnectionManager.new(user: User.current, oauth_client:)
+            connection_manager.authorization_state
```

Comment:
This can be rewritten to

```
::OAuthClients::ConnectionManager
  .new(user: User.current, oauth_client: storage.oauth_client)
  .authorization_state
```

It avoids assigning two variables and I'd argue it is simpler to read.

Reply:
Yes sure I had it before that. But wanted to find out what style is preferred. ;)